Index: SegmentValidator/public/src/ValidateMP4.cpp
===================================================================
--- SegmentValidator/public/src/ValidateMP4.cpp	(revision 100)
+++ SegmentValidator/public/src/ValidateMP4.cpp	(working copy)
@@ -421,7 +421,7 @@
 	vg.inOffset = 0;
 	err = fseek(infile, 0, SEEK_END);
 	if (err) goto bail;
-	UInt64 tttemp = ftell(infile);
+//	UInt64 tttemp = ftell(infile);
 	vg.inMaxOffset = inflateOffset(ftell(infile));
 	if (vg.inMaxOffset < 0) {
 		err = vg.inMaxOffset;
